connector/proto/src/main/scala/org/apache/spark/sql/proto/CatalystDataToProto.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.proto
```

Comment:
Regd naming: `to_proto` or `to_protobuf`?
I was thinking the latter. Protobuf normally refers to the actual protobuf messages (serialized or not). These APIs deal with protobufs. _proto_ is more generic and refers to many things: proto files that define the protobuf schemas, etc. 
IOW, `proto` is the schema/definition and `protobuf` is the instance of actual messages.  WDYT? 
Of course naming is subjective.

Reply:
That makes sense. I am assuming that this would be a wholesale change wherever proto appears ?

Reply:
Yes, in most places. Especially the interface. Thank you for the flexibility.